independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/config/ExtensionsFilter.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.analytics.config;
+
+import java.util.List;
+
+import io.quarkus.devtools.messagewriter.MessageWriter;
+
+public class ExtensionsFilter {
+    private static final List<String> AUTHORIZED_GROUPS = List.of(
+            "io.quarkus",
+            "io.quarkiverse",
+            "org.apache.camel.quarkus",
+            "io.debezium",
+            "org.drools",
+            "org.optaplanner",
+            "org.amqphub.quarkus",
+            "com.hazelcast",
+            "com.datastax.oss.quarkus");
```

Comment:
OK, so you need to clarify the list in the documentation because we are not doing what we documented.

Reply:
These are group Ids of extensions bundled on the platform.